install_files/ansible-base/roles/backup/files/0.3_collect.py

```diff
@@ -56,7 +56,8 @@ def encrypt_zip_file(zf_fn):
     gpg = gnupg.GPG(binary='gpg2', homedir=config.GPG_KEY_DIR)
     e_fn = '{}.gpg'.format(zf_fn)
 
-    gpg.encrypt_file(zf_fn, config.JOURNALIST_KEY, always_trust='True', output=e_fn)
+    stream = open(zf_fn, "rb")
+    gpg.encrypt_file(stream, config.JOURNALIST_KEY, always_trust='True', output=e_fn)
```

Comment:
Investigated this a bit. Looks like the implementation of `encrypt_file` in python-gnupg in 1.2.5 does not do what is advertised on the tin (i.e. in the docstring). Don't want to take the time to explain everything here, but basically - this function is totally broken and does not work if a filename is passed as the first parameter. Using a stream (somehow) works.